lms/static/js/spec/edxnotes/views/shim_spec.js

```diff
@@ -233,5 +233,25 @@ define([
                 expect(mockViewer.element.appendTo).toHaveBeenCalledWith(annotators[0].wrapper);
             });
         });
+
+        describe('TagsPlugin', function () {
```

Comment:
@cahrens thanks.  I would like to get a better understanding of how we can add more accessibility related tests.  Knowing when there are limitations is helpful.  Would love to talk more about this at some point (bok choy vs unit test for this use case).